FoodBookApp/UI/Views/CreateReview/CreateReview2View.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  CreateReview2View.swift
+//  FoodBookApp
+//
+//  Created by Laura Restrepo on 28/02/24.
+//
+
+import Foundation
+import SwiftUI
+import PhotosUI
+
+struct CreateReview2View: View {
+    let categories: [String]
+    @State private var pickerItem: PhotosPickerItem?
+    @State private var selectedImage: UIImage?
+    
+    @State private var showSheet: Bool = false
+    @State private var showImagePicker: Bool = false
+    @State private var sourceType: UIImagePickerController.SourceType = .camera
+    
+    @State private var addPhotoText: String = "Add a photo..."
+    @State private var cleanliness: Int = 0
+    @State private var waitingTime: Int = 0
+    @State private var service: Int = 0
+    @State private var foodQuality: Int = 0
+    @State private var reviewTitle: String = ""
+    @FocusState private var reviewTitleIsFocused: Bool
+    @State private var reviewBody: String = ""
+    @FocusState private var reviewBodyIsFocused: Bool
+    let customGray = Color(red: 242/255, green: 242/255, blue: 242/255)
```

Comment:
Same as above

Reply:
Definitely, and as you say it should be an issue later because I also used these colors in another branch so it's better to avoid conflicts. Thanks for the feedback!